TACC/Core-CMS-Custom
# feat: ctrn_assets (placeholder files)

## Overview

Assets for CTRN (Childhood Trauma Research Network).

## Status

> [!WARNING]
> Pending TACC/Core-CMS setup on server.

## Related

- [#ctrn-dev "CTRN Progress" list (internal, private)](https://tacc-team.slack.com/lists/T03EBR1EB/F07N0UKRSQ6)

## Changes

- **added** image
- **added** stylesheet

## Testing

1. Visit http://ctrn-web.tacc.utexas.edu/.
2. Verify logo is visible.
3. Verify stylesheet is loaded.

## UI

…
